addons/hr/models/hr_employee.py

```diff
@@ -160,14 +160,16 @@ def _compute_avatar_128(self):
         super()._compute_avatar_128()
 
     def _compute_avatar(self, avatar_field, image_field):
+        employee_wo_user_or_image_ids = []
```

Comment:
Can you reword a little bit your commit message: 
```txt
[IMP] hr: add default avatar of employee

Before this commit, when the user creates new resource and employee on 
the fly in many2one field, the default avatar of Initial name should be 
set instead of placeholder (grey icon). The reason why a grey icon is
displayed is because the `_compute_avatar` for employee set that icon
when avatar or image is not found.

This commit add default avatar of initial name when new resource and employee is
created.

task-3251689
```

Reply:
@xavierbol ,
Changes Done.  Thank You!!